lib/modules/dosomething/dosomething_gladiator/dosomething_gladiator.install

```diff
@@ -90,3 +92,20 @@ function dosomething_gladiator_update_7002(&$sandbox) {
     }
   }
 }
+
+/**
+ * Update response values to be a long text field, as the response is LONG.
+ */
+function dosomething_gladiator_update_7003(&$sandbox) {
+  $field = 'response_values';
+  $table = 'dosomething_gladiator_failed_task_log';
+  // Update the field to a large text.
+  db_change_field($table, $field, $field, array(
+    'description' => 'The JSON response from Gladiator',
+    'type' => 'text',
+    'length' => '500000',
+    'not null' => FALSE,
+    'default' => NULL,
```

Comment:
AH, no that's an excellent question! it's going to take me some digging to find the issue, but we _didn't_ do this for an update in `dosomething_reportback.install`
it seems when you have multiple webheads after the update hook runs, it'll use the `schema` to define the table.
so I remember I changed something to be default null, and found it wasn't working until the schema was also updated.